### PR TITLE
wakatime: Explicit `checkPhase`

### PR DESCRIPTION
**Pull request by markus1189**
The actual tests write to the filesystem and lead to errors, so I replaced them with a different sanity check for the version

Related #32244

### CODE REVIEW

**Comment by jtojnar** [comment]
The write errors can be fixed by adding

```
  preCheck = ''
    export HOME=$(mktemp -d)
  '';
```

**Comment by markus1189** [comment]
Ah nice :sunglasses: 

But unfortunately the tests run and fail with assertion errors :(

File: pkgs/tools/misc/wakatime/default.nix [hunk 1]
```diff
@@ -1,7 +1,8 @@
 { stdenv, buildPythonApplication, fetchFromGitHub }:
 
 buildPythonApplication rec {
-  name = "wakatime-${version}";
+  baseName = "wakatime";
```
Comment (FRidh) [inline pkgs/tools/misc/wakatime/default.nix]: pname

File: pkgs/tools/misc/wakatime/default.nix [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+  name = "${baseName}-${version}";
```
Comment (FRidh) [inline pkgs/tools/misc/wakatime/default.nix]: name can be dropped

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/wakatime/default.nix (hunk 2 above)
Comment (markus1189) [reply pkgs/tools/misc/wakatime/default.nix]: Does it default to `${pname}-${version}`?

Comment (FRidh) [reply pkgs/tools/misc/wakatime/default.nix]: In ` buildPython*`, yes.

**Comment by FRidh** [comment]
> But unfortunately the tests run and fail with assertion errors :(

And they're not fixable?

**Comment by markus1189** [comment]
I don't know, I did not test :D

**Comment by jtojnar** [comment]
Tests were disabled in https://github.com/NixOS/nixpkgs/pull/33917

**Pull request closed without merging**